### PR TITLE
dbs-cli: several refactor in cmd tool

### PR DESCRIPTION
**Pull request by studychao**
1. Add all creation related args under Create command and all update related args under Update command.

2. catagorize different args type to make more sense, like putting all cpu related commands under CpuArgs instead of putting them everywhere without order.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,13 +74,13 @@ Create virtio-net devices.
...
 ```
-./dbs-cli \
+./dbs-cli create \
   --log-file dbs-cli.log --log-level ERROR \
   --kernel-path ~/path/to/kernel/vmlinux.bin \
   --rootfs ~/path/to/rootfs/bionic.rootfs.ext4 \
   --boot-args "console=ttyS0 console=ttyS1 earlyprintk=ttyS1 tty0 reboot=k debug panic=1 pci=off root=/dev/vda1" \
   --virnets "[{\"iface_id\":\"eth0\",\"host_dev_name\":\"tap0\",\"num_queues\":2,\"queue_size\":0,\"guest_mac\":\"43:2D:9C:13:71:48\",\"allow_duplicate_mac\":true}]" \
-  create;
+  ;
```
Comment (justxuewei) [inline README.md]: Move to previous line?

File: README.md [hunk 2]
```diff
@@ -89,13 +89,13 @@ Create virtio-blk devices.
...
 ```
-./dbs-cli \
+./dbs-cli create\
   --log-file dbs-cli.log --log-level ERROR \
   --kernel-path ~/path/to/kernel/vmlinux.bin \
   --rootfs ~/path/to/rootfs/bionic.rootfs.ext4 \
   --boot-args "console=ttyS0 console=ttyS1 earlyprintk=ttyS1 tty0 reboot=k debug panic=1 pci=off root=/dev/vda1" \
   --virblks '[{"drive_id":"testblk","device_type":"RawBlock","path_on_host":"/path/to/test.img","is_root_device":false,"is_read_only":false,"is_direct":false,"no_drop":false,"num_queues":1,"queue_size":1024}]' \
-  create;
+  ;
```
Comment (justxuewei) [inline README.md]: Ditto.

File: README.md [hunk 3]
```diff
@@ -117,9 +117,9 @@ Create hot-plug virtio-net devices via API Server:
 
 ```
 sudo ./dbs-cli  \
-  --api-sock-path [socket path]
+  --api-sock-path [socket path] update
   --hotplug-virnets "[{\"iface_id\":\"eth0\",\"host_dev_name\":\"tap0\",\"num_queues\":2,\"queue_size\":0,\"guest_mac\":\"43:2D:9C:13:71:48\",\"allow_duplicate_mac\":true}]" \
```
Comment (justxuewei) [inline README.md]: Remove `\` at the end?

File: README.md [hunk 4]
```diff
@@ -129,11 +129,13 @@ Create hot-plug virtio-blk devices via API Server:
 
 ```
 sudo ./dbs-cli  \
-  --api-sock-path [socket path]
+  --api-sock-path [socket path] update
   --hotplug-virblks '[{"drive_id":"testblk","device_type":"RawBlock","path_on_host":"/path/to/test.img","is_root_device":false,"is_read_only":false,"is_direct":false,"no_drop":false,"num_queues":1,"queue_size":1024}]' \
```
Comment (justxuewei) [inline README.md:130]: Ditto.

File: src/parser/args.rs [hunk 5]
```diff
@@ -237,12 +174,14 @@ The type of it is an array of BlockDeviceConfigInfo, e.g.
...
 
-/// Config boot source including rootfs file path
-#[derive(Args, Debug, Deserialize, Serialize, Clone)]
-#[clap(arg_required_else_help = true)]
-pub struct BootArgs {
+    // feature for log
+    #[clap(long, value_parser, default_value = "dbs-cli.log", display_order = 1)]
+    pub log_file: String,
+
+    #[clap(long, value_parser, default_value = "Info", display_order = 1)]
```
Comment (justxuewei) [inline src/parser/args.rs]: Is "Debug" a better default value? The reason is that dbs-cli is a debug tool for Dragonball.

File: README.md [hunk 6]
```diff
@@ -129,11 +129,13 @@ Create hot-plug virtio-blk devices via API Server:
 
 ```
 sudo ./dbs-cli  \
-  --api-sock-path [socket path]
+  --api-sock-path [socket path] update
```
Comment (justxuewei) [inline README.md]: For update subcommand, is it necessary to have a `hotplug` prefix? It is a quite burden on typing it from my perspective.

File: README.md [hunk 7]
```diff
@@ -15,22 +15,22 @@ See all options:
 A simple example:
 
 ```bash
-./dbs-cli \
+./dbs-cli create\
```
Comment (justxuewei) [inline README.md]: Add a space behind "create": `./dbs-cli create \`.

File: README.md [hunk 8]
```diff
@@ -15,22 +15,22 @@ See all options:
...
   --kernel-path ~/path/to/kernel/vmlinux.bin \
   --rootfs ~/path/to/rootfs/rootfs.dmg \
-  --boot-args "console=ttyS0 console=ttyS1 earlyprintk=ttyS1 tty0 reboot=k debug panic=1 pci=off root=/dev/vda1" create ;
+  --boot-args "console=ttyS0 console=ttyS1 earlyprintk=ttyS1 tty0 reboot=k debug panic=1 pci=off root=/dev/vda1" ;
```
Comment (justxuewei) [inline README.md]: Please remove the space at the end.

File: README.md [hunk 9]
```diff
@@ -15,22 +15,22 @@ See all options:
...
 ```
 
 Set the log level and log file:
 
 > The log-level argument is case-insensitive: ErrOR and InFO are valid.
 
 ```bash
-./dbs-cli \
+./dbs-cli create\
```
Comment (justxuewei) [inline README.md]: Ditto.

File: README.md [hunk 10]
```diff
@@ -44,12 +44,12 @@ Create a vsock console (communication with sock file)
 > Otherwise, `dbs-cli` will create a vsock console with a sock file, namely the value of `serial-path`.
 
 ```
-./dbs-cli \
+./dbs-cli create\
```
Comment (justxuewei) [inline README.md]: Ditto.

File: README.md [hunk 11]
```diff
@@ -60,12 +60,12 @@ Create a virtio-vsock tunnel for Guest-to-Host communication.
 > specified with the `--vsock` parameter.
 
 ```
-./dbs-cli \
+./dbs-cli create\
```
Comment (justxuewei) [inline README.md]: Ditto.

File: README.md [hunk 12]
```diff
@@ -89,13 +89,13 @@ Create virtio-blk devices.
 > format of JSON.
 
 ```
-./dbs-cli \
+./dbs-cli create\
```
Comment (justxuewei) [inline README.md]: Ditto.

**Review by justxuewei — COMMENTED** [review]
Thanks @studychao! A few comments here.

File: README.md (hunk 1 above)
Comment (studychao) [reply README.md]: done and I remove all redundant `;`

File: README.md (hunk 2 above)
Comment (studychao) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (studychao) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (studychao) [reply README.md:130]: done

File: src/parser/args.rs (hunk 5 above)
Comment (studychao) [reply src/parser/args.rs]: changed.

File: README.md (hunk 6 above)
Comment (studychao) [reply README.md]: all `hotplug` prefix are removed,

File: README.md (hunk 7 above)
Comment (studychao) [reply README.md]: done

File: README.md (hunk 8 above)
Comment (studychao) [reply README.md]: done

File: README.md (hunk 9 above)
Comment (studychao) [reply README.md]: done

File: README.md (hunk 10 above)
Comment (studychao) [reply README.md]: done

File: README.md (hunk 11 above)
Comment (studychao) [reply README.md]: done

File: README.md (hunk 12 above)
Comment (studychao) [reply README.md]: done

**Review by justxuewei — APPROVED** [review]
Lgtm, thanks @studychao!

**Pull request merged**